shared/common-adapters/input.native.js

```diff
@@ -148,7 +148,7 @@ class Input extends Component<void, Props, State> {
       fontSize: _headerTextStyle.fontSize,
       lineHeight: _lineHeight,
       backgroundColor: globalColors.transparent,
-      flex: 1,
+      flexGrow: 1,
```

Comment:
with that breaking change in RN we likely have to do changes like this to fix the squished views. This updated version of RN actually gives you a redbox vs rendering something it knows is invalid